source/mainnet/net/resources/release-notes.rst

```diff
@@ -11,6 +11,15 @@ Release Notes - Testnet
 
    To subscribe to updates on the Testnet status page click **Subscribe** to get all updates or click **Get updates** to choose to get all updates or only updates for specific products.
 
+May XX, 2023
+
+|mw-gen2| for Android 1.1.8
+-----------------------------------------
+
+The terms and conditions have been updated and now point to a link where you can read the updated terms and conditions before accepting. Additionally, when the terms and conditions have been updated, you will be prompted to accept them again.
```

Comment:
```suggestion
The prompt to review the terms and conditions has been updated, and it now points to a link where you can read the newest version of the terms and conditions before accepting. Additionally a new prompt will be shown in the wallet, if the terms and conditions are updated, so it no longer happens only after updating the application. 
```
I don't know if this is better, it just felt a little bit like it was only the T&C that had been updated and not the UI as well.